### DISCUSSION TITLE
Bare-metal IPI install of 4.17.0-0.okd-scos-2024-06-25-032248

### DISCUSSION THREAD

**Discussion by bernhardloos** [okd-project/okd#1956] [Pre-Release Testing]
It worked in general and was way faster then 4.15-FCOS but I have a bug that looks a lot like this: https://issues.redhat.com/browse/OCPBUGS-32304
OCPBUGS-32304 was supposed to be fixed in [4.17.0-0.nightly-2024-06-05-034505](https://amd64.ocp.releases.ci.openshift.org/releasestream/4.17.0-0.nightly/release/4.17.0-0.nightly-2024-06-05-034505) so I'm not sure if this is a different issue or if OKD is missing some commits somewhere.
metal3-ironic-inspector never came up as far as I can tell, not special steps were needed to reproduce the problem.

**Comment by eb4x** [comment]
I did not get that version working on my baremetal setup. Two things that stood out were bootstrap node starts out as fedora-coreos, then release-image-pivot to stream-coreos. And when the control-plane nodes boot up into the provisioning stage, they're given RHCOS, which probably should not happen?

They never get out of the provisioning stage.

**Reply by bernhardloos** [reply]
How did you install it? My CP nodes are on CentOS Stream CoreOS 416.9.202406111032-0

**Reply by eb4x** [reply]
After some additional investigation, the problem with nodes getting stuck during provisioning was the `hctl` value I had supplied to `rootDeviceHints`. Its value had changed for some reason. (Possibly difference in kernels?)

The nodes are now running Stream CoreOS 416.9.202406111032-0 here aswell.
